pom.xml

```diff
@@ -696,9 +711,14 @@
                 <artifactId>asm-commons</artifactId>
                 <version>${asm.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.scala-lang</groupId>
+                <artifactId>scala-library</artifactId>
+                <version>${scala.version}</version>
```

Comment:
Why do you need scala as a compile dependency?

Reply:
If it's to override version, it's better to use dependency management

Reply:
Does dependencyManagement override transitive dependencies?

Reply:
Yes, that's the best part about it!

Reply:
Yeah, made this change and tested it.